src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+
+import java.util.List;
+
+
```

Comment:
There is a redundant blank line here (and also below in the rest of your code, e.g. line 18). Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.